connectors/sources/sharepoint.py

```diff
@@ -44,15 +44,18 @@
 LIST_ITEM = "list_item"
 ATTACHMENT_DATA = "attachment_data"
 DOCUMENT_LIBRARY = "document_library"
+LIST_BY_TITLE = " list_by_title"
+TIKA_SUPPORTED_FILETYPES.append(".aspx")
 
 URLS = {
     PING: "{host_url}/sites/{site_collections}/_api/web/webs",
     SITES: "{host_url}{parent_site_url}/_api/web/webs?$skip={skip}&$top={top}",
     LISTS: "{host_url}{parent_site_url}/_api/web/lists?$skip={skip}&$top={top}&$expand=RootFolder&$filter=(Hidden eq false)",
     ATTACHMENT: "{host_url}{value}/_api/web/GetFileByServerRelativeUrl('{file_relative_url}')/$value",
-    DRIVE_ITEM: "{host_url}{parent_site_url}/_api/web/lists(guid'{list_id}')/items?$select=Modified,Id,GUID,File,Folder&$expand=File,Folder&$top={top}",
+    DRIVE_ITEM: "{host_url}{parent_site_url}/_api/web/lists(guid'{list_id}')/items?$select=*&$expand=File,Folder&$top={top}",
```

Comment:
For fetching the site pages content for SharePoint sever we are getting all the data in a field called as `WikiField` but this field is missing for SharePoint online and if I added `WikiField` here for server then it will throw error for online. That's why I replace it with * .Otherwise we can also manage it with two different URL where the selecting list of fields will be different for server and online

Reply:
@vidok I am attaching the response for using * as well as without * , the only difference is that now we are getting these extar field in the response: `FileSystemObjectType`,`ServerRedirectedEmbedUri`,`ServerRedirectedEmbedUrl`,`ContentTypeId`,`Created`,`AuthorId`,`EditorId`,`OData__CopySource`,`CheckoutUserId`,`OData__ColorTag`,`OData__UIVersionString`,`ComplianceAssetId`,`Title`

But If we need to go with limited fields then we need to execute same API with different list of selected fields  for server and online.. As we need WikiField to be present for server to extract the aspx content

Reply:
[response.txt](https://github.com/elastic/connectors-python/files/11285742/response.txt)